cmd/utils/serverConfigVars.go

```diff
@@ -31,7 +31,7 @@ func SetConfigVars(serverConfigFilePath string) error {
         mainConfig := GetServerConfigFromFile(serverConfigFilePath)
         Logln(LogPrefixInfo + " reading '" + serverConfigFilePath + "'")
 
-        RESTAPIBase = mainConfig.Host + ":" + mainConfig.Port + "/" + Context + "/"
+        RESTAPIBase = mainConfig.Url + ":" + mainConfig.Port + "/" + Context + "/"
```

Comment:
Shall we rename this variable too since it is not in the correct style?

Reply:
are you referring to RESTAPIBase ?